zerver/worker/queue_processors.py

```diff
@@ -480,6 +482,17 @@ def get_bot_api_client(self, user_profile):
 
     def get_bot_handler(self, service):
         # type: (Service) -> Any
+
+        # Convert unicode service.name to string.
+        str_service_name = unicodedata.normalize('NFKD', service.name).encode('ascii', 'ignore')
```

Comment:
I guess we'll have to change it in the code path that creates Service object. Looking into it!